### PR TITLE
Added test for send-targets script

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

Added initial test for send-targets ...

**Why**:

We have no tests for our scripts

**How**:

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gvm-tools/blob/master/CHANGELOG.md) Entry
- [ ] Documentation

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -4,3 +4,4 @@ omit =
     tests/*
 source =
     gvmtools
+    scripts
```
Comment (y0urself) [inline .coveragerc:7]: This did not work as expected. @bjoernricks how can I add the scripts directory to the coverage?

Comment (bjoernricks) [reply .coveragerc:7]: I don't know. Did you take a look at the docs https://coverage.readthedocs.io/en/latest/ ?

**Review by bjoernricks — COMMENTED** [review]
Maybe https://www.dev2qa.com/how-to-import-a-python-module-from-a-python-file-full-path/ could work to load the script from `foo.gmp.py`?

File: .coveragerc (hunk 1 above)
Comment (y0urself) [reply .coveragerc:7]: The problem is, that this PR is not running codecov ...

**Comment by y0urself** [comment]
> Maybe https://www.dev2qa.com/how-to-import-a-python-module-from-a-python-file-full-path/ could work to load the script from `foo.gmp.py`?

That indeed worked ... so we can do it like this ...

File: tests/scripts/__init__.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# hacky ... hacky ... loading a script ...
+def load_script(path: Union[str, Path], script_name: str):
+    spec = spec_from_file_location(
+        script_name, '{}/{}.gmp.py'.format(path, script_name)
+    )
+    script = module_from_spec(spec)
+    spec.loader.exec_module(script)
+
+    return script
```
Comment (bjoernricks) [inline tests/scripts/__init__.py:36]: I don't think this is hacky. You could also look at https://github.com/greenbone/gvm-tools/blob/e04cda21b7f3065ea79c95140aef051b86f99184/gvmtools/helper.py#L132 how the scripts are actually executed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ $ cd gvm-tools && git log
 * Added `pretty_print` to `pyshell` by default, so it does not need to be manually imported [#305](https://github.com/greenbone/gvm-tools/pull/305)
 * Added tests for `helper` module [#310](https://github.com/greenbone/gvm-tools/pull/310)
 * Added tests for `parser` module [#311](https://github.com/greenbone/gvm-tools/pull/311)
+* Added test for `script/send-target.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
```
Comment (bjoernricks) [inline CHANGELOG.md:22]:
```suggestion
* Added test for `script/send-targe.gmp.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
```

File: tests/scripts/__init__.py (hunk 2 above)
Comment (y0urself) [reply tests/scripts/__init__.py:36]: But I think we are fine with this function to load the scripts for the tests?

Comment (bjoernricks) [reply tests/scripts/__init__.py:36]: Of course I just find the comment misleading

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ $ cd gvm-tools && git log
 * Added `pretty_print` to `pyshell` by default, so it does not need to be manually imported [#305](https://github.com/greenbone/gvm-tools/pull/305)
 * Added tests for `helper` module [#310](https://github.com/greenbone/gvm-tools/pull/310)
 * Added tests for `parser` module [#311](https://github.com/greenbone/gvm-tools/pull/311)
+* Added test for `script/send-targe.gmp.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
```
Comment (bjoernricks) [inline CHANGELOG.md:22]:
```suggestion
* Added test for `script/send-target.gmp.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
```

Comment (bjoernricks) [reply CHANGELOG.md:22]: Sorry I guess i've introduced this typo

Comment (y0urself) [reply CHANGELOG.md:22]: yeah. but I have just commited your suggestion ...

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**